src/cargo/util/process_builder.rs

```diff
@@ -129,11 +131,11 @@ impl ProcessBuilder {
     }
 }
 
-pub fn process<T: AsRef<OsStr>>(cmd: T) -> CargoResult<ProcessBuilder> {
+pub fn process<T: AsRef<OsStr>, U: AsRef<OsStr>>(cmd: T, cwd: U) -> CargoResult<ProcessBuilder> {
```

Comment:
Hm, well I reasoned that since this is a internal helper API for cargo, and all uses in cargo should respect `Config::cwd`, that making it outright required would catch potential bugs.